tests/python/pants_test/pants_run_integration_test.py

```diff
@@ -124,6 +124,20 @@ def has_python_version(cls, version):
     except OSError:
       return False
 
+  @classmethod
+  def python_interpreter_path(cls, version):
```

Comment:
The `has_python_version` method above should probably chain to this new method, returning `python_interpreter_path(version) is not None`.